integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6323,32 +6323,38 @@ msgstr "Benutzer {} wurde erfolgreich von dieser Region entfernt."
 
 #: cms/views/users/region_user_form_view.py:56
 #: cms/views/users/region_user_form_view.py:131
-#: cms/views/users/user_form_view.py:54 cms/views/users/user_form_view.py:135
+#: cms/views/users/user_form_view.py:55 cms/views/users/user_form_view.py:144
 msgid "Pending account activation"
 msgstr "Aktivierung des Benutzerkontos noch ausstehend"
 
 #: cms/views/users/region_user_form_view.py:112
-#: cms/views/users/user_form_view.py:117
+#: cms/views/users/user_form_view.py:126
 msgid "User \"{}\" was successfully created"
 msgstr "Benutzer \"{}\" wurde erfolgreich erstellt"
 
 #: cms/views/users/region_user_form_view.py:120
-#: cms/views/users/user_form_view.py:125
+#: cms/views/users/user_form_view.py:134
 msgid "User \"{}\" was successfully saved"
 msgstr "Benutzer \"{}\" wurde erfolgreich gespeichert"
 
 #: cms/views/users/user_actions.py:40
 msgid "User was successfully deleted"
 msgstr "Benutzer wurde erfolgreich gelöscht"
 
-#: cms/views/users/user_form_view.py:100
+#: cms/views/users/user_form_view.py:104
 msgid ""
 "A user has to be either staff/superuser or needs to be restricted to at "
 "least one region."
 msgstr ""
 "Ein Benutzer muss entweder Mitarbeiter/Super-Administrator sein, oder "
 "mindestens einer Region zugewiesen sein."
 
+#: cms/views/users/user_form_view.py:110
+msgid "Superuser permissions need to be set by another superuser."
+msgstr ""
+"Super-Administratorrechte müssen von einem anderen Superadministrator "
+"vergeben werden."
```

Comment:
```suggestion
"Administratorrechte müssen von einem anderen Administrator vergeben werden."
```
Since we only have one level of admins, I'd go with "superuser" in English and "Administrator" in German. ("Super-Administrator" sounds a bit like there are different, normal "Administrators" out there...)
I'm aware that there are inconsistencies in our existing translations, so if you want to, you can also fix these, but they are not necessarily part of this issue.